src/System.Security.Cryptography.Cng/pkg/System.Security.Cryptography.Cng.pkgproj

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <ProjectReference Include="..\ref\System.Security.Cryptography.Cng.csproj">
+      <SupportedFramework>net461;netcoreapp2.0;uap10.1</SupportedFramework>
+    </ProjectReference>
+    <ProjectReference Include="..\src\System.Security.Cryptography.Cng.csproj" />
+    
+    <!-- All elements from previous packages that will be included in the newly built package -->
+    <HarvestIncludePaths Include="ref/net46;lib/net46;runtimes/win/lib/net46" />
+    <HarvestIncludePaths Include="ref/netstandard1.3" />
```

Comment:
I suspect these will fail with the latest buildtools because there isn't a RID-less netstandard impl.  You can have a look at an example of I fixed this for https://github.com/dotnet/corefx/blob/af74a86c3003d487e981655283488f8b03829b29/src/System.ServiceProcess.ServiceController/pkg/System.ServiceProcess.ServiceController.pkgproj#L9-L17.